### DISCUSSION TITLE
403 on assets on login page

### DISCUSSION THREAD

**Discussion by mbroekman** [requarks/wiki#3173] [Help / Questions] [answered]
Hi. I try to use own images on the login page but get on both images a 403. The images are added in Wiki and selected in the admin area. How can i solve this ?

OS: Ubuntu 20.04.2 LTS and Docker
Wiki.js version: 2.5.170 ( docker requarks/wiki:2)
Database engine: PostgreSQL 11.11

![2021-03-04 14_55_02-Window](https://user-images.githubusercontent.com/4540115/109974301-b298ee80-7cf9-11eb-9246-67b737e2e3da.png)

Regards,
Machiel

**Comment by NGPixel** [comment, marked as answer]
You need to add read access to the path on the Guests group. So if you uploaded your images to /images/login.jpg, you need to create a rule with `read:assets` permission on the path `/images`

**Comment by mbroekman** [comment]
Thanks, that solved it.
